charmcraft/commands/build.py

```diff
@@ -240,8 +240,9 @@ def process(self, parsed_args):
     def validate_from(self, arg):
```

Comment:
Should we add annotations to be clear that arg isn't a str?

Reply:
Nothing is a string! :p Anyway, we could bring annotations in the future for everything, but that's a separate talk,,,